src/util/json_utils.js

```diff
@@ -67,5 +67,19 @@ class JsonObjectWrapper {
 
 }
 
+async function format_and_send_event(event) {
+    const event_json = "{ \"code\":\"" + event.code + "\"," +
```

Comment:
Can't we create a js object and simply `JSON.stringify(value)`?

Reply:
agree. this can be simply converted to 
```javascript
    const event_json = JSON.stringify(event);
```

Reply:
Also, I don't think it belongs in `json_utils.js`. This module was mainly used to handle jsons stored in files.
Maybe it makes sense to move this implementation to debug_module